decidim-core/app/packs/stylesheets/decidim/_redesigned_hero.scss

```diff
@@ -0,0 +1,9 @@
+.hero{
+  @apply h-[370px] md:h-[656px] py-24 flex flex-col justify-center items-center gap-12 md:gap-16;
```

Comment:
should we avoid fixed dimensions?

Reply:
Heroes are always "special". Initially I went with 50vh, but it wasn't convinced me, so I rollback. Anyways, I realized about another potential issues (as very looooong titles) I didn't face yet, so I'll back to test more edge cases

Reply:
Let's go with aspect-video (16/9) by default. This way, users can have very simple and standard guidelines on the proportions their images should have.

Reply:
https://tailwindcss.com/docs/aspect-ratio

Reply:
Done

Reply:
Mmm... because of the fixed column widths we are not taking full advantage of the aspect-ratio. But lets leave it like this for the moment.